druntime/src/core/time.d

```diff
@@ -2811,14 +2839,14 @@ struct TickDuration
 
     version (CoreUnittest) unittest
     {
-        assert(zero == TickDuration(0));
-        assert(TickDuration.max == TickDuration(long.max));
-        assert(TickDuration.min == TickDuration(long.min));
-        assert(TickDuration.min < TickDuration.zero);
-        assert(TickDuration.zero < TickDuration.max);
-        assert(TickDuration.min < TickDuration.max);
-        assert(TickDuration.min - TickDuration(1) == TickDuration.max);
-        assert(TickDuration.max + TickDuration(1) == TickDuration.min);
+        assert((zero == TickDuration(0)) == true);
```

Comment:
What are these `== true` additions for?

Reply:
ping @schveiguy ?

Reply:
Oof, I missed this comment!

> Note all the changes to assert statements are to work around https://issues.dlang.org/show_bug.cgi?id=23800

That is why.

Reply:
ping @dkorpel in case the context isn't clear 2 weeks later. Would love to see this merged for the next release.